gubernator/view_base.py

```diff
@@ -111,13 +111,13 @@ def memcache_memoize(prefix, expires=60 * 60, neg_expires=60):
     namespace = os.environ['CURRENT_VERSION_ID']
     def wrapper(func):
         @functools.wraps(func)
-        def wrapped(arg):
-            key = '%s%s' % (prefix, arg)
+        def wrapped(*args):
```

Comment:
It's just a little cleanup. "Fix memcache_memoize decorator to handle multi-argument functions."